.lando.yml

```diff
@@ -39,6 +39,10 @@ services:
     # User random static high port for database connection.
     portforward: 34567
   appserver:
+    build_as_root:
+      # To be able to run ESLint with GrumPHP, we need Node.js on appserver.
+      - "apt-get update"
+      - "apt-get install -y nodejs"
```

Comment:
I would avoid such things in a template - they slow down the build time. Could we add `nodejs` to the Silta PHP image instead?